helm-charts/splunk-otel-collector/templates/_helpers.tpl

```diff
@@ -438,6 +438,14 @@ Whether clusterReceiver should be enabled
 {{- end -}}
 
 
+{{/*
+Whether persistentQueue should be enabled
+*/}}
+{{- define "splunk-otel-collector.persistentQueueEnabled" }}
+{{- .Values.splunkPlatform.sendingQueue.persistentQueue.enabled -}}
+{{- end }}
```

Comment:
No need to introduce a helper for this. Just use `.Values.splunkPlatform.sendingQueue.persistentQueue.enabled`